samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/Application.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.multilingual;
+
+import com.microsoft.bot.builder.ConversationState;
+import com.microsoft.bot.builder.Storage;
+import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.integration.AdapterWithErrorHandler;
+import com.microsoft.bot.integration.BotFrameworkHttpAdapter;
+import com.microsoft.bot.integration.Configuration;
+import com.microsoft.bot.integration.spring.BotController;
+import com.microsoft.bot.integration.spring.BotDependencyConfiguration;
+import com.microsoft.bot.sample.multilingual.translation.TranslationMiddleware;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+
+/**
+ * This is the starting point of the Sprint Boot Bot application.
+ *
+ * This class also provides overrides for dependency injections. A class that
+ * extends the {@link com.microsoft.bot.builder.Bot} interface should be
+ * annotated with @Component.
+ *
+ * @see MultiLingualBot
+ */
+@SpringBootApplication
+
+// Use the default BotController to receive incoming Channel messages. A custom
+// controller could be used by eliminating this import and creating a new
+// RestController.
+// The default controller is created by the Spring Boot container using
+// dependency injection. The default route is /api/messages.
+@Import({BotController.class})
+
+public class Application extends BotDependencyConfiguration {
+    public static void main(String[] args) {
```

Comment:
On my checkstyle, it asks for a javadoc here

Reply:
Just to confirm, you checkstyle is using the rules of ms?

Reply:
Its pointing to the local xml

Reply:
I don't see another Application with comments. Let's wait for feedback of tracy here